### DISCUSSION TITLE
Issue template chooser for creation dialog within a project

### DISCUSSION THREAD

**Discussion by thinkh** [community/community#46167] [Projects and Issues]
Thanks for the most recent feature to [create a new issues in a modal dialog within a project board or table](https://github.blog/changelog/2023-01-26-github-projects-january-26th-update/#create-issues-in-a-snap-with-the-new-issue-creation-dialog). This is very helpful!

It would be great to have an additional drop-down or step to choose an issue template.

For example, if I want to create a bug report I'd still need to go to the regular issue creation form where I can select a template in the first step.

**Comment by rileybroughten** [comment]
Thanks @thinkh for this feedback! Tagging @j0siepy for feedback on the new modal dialog relating to templates 👍

**Comment by marikaner** [comment]
I'd love this feature too. We create all issues based on templates. As long as those are not selectable from the dialog, we basically cannot use the dialog (unless we copy paste the template manually).

**Reply by joaomas** [reply]
> I'd love this feature too. We create all issues based on templates. As long as those are not selectable from the dialog, we basically cannot use the dialog (unless we copy paste the template manually).

Me too

**Comment by YaroShkvorets** [comment]
This is a huge blocker. Makes creating issues from the project board hard.
